tasks/service-runner/app/workers/pull-worker.ts

```diff
@@ -226,10 +206,7 @@ export default class PullWorker implements Worker {
     let primeCount = 0;
     while (!isPrimed && primeCount < maxPrimeRetries) {
       try {
-        if (env.harmonyService.includes('harmonyservices/query-cmr')) {
-          // called this way to support sinon spy
-          await exportedForTesting._primeCmrService();
-        } else {
+        if (!env.harmonyService.includes('harmonyservices/query-cmr')) {
```

Comment:
There is no need to prime the queryCmr service anymore so I removed it because it was putting bogus error messages in the log.